README.md

```diff
@@ -81,15 +81,15 @@ We are excited to work alongside you, our amazing community, to build and enhanc
 
 The easiest way to communicate with the team is via GitHub issues.
 
-Please file new issues, feature requests, and suggestions but **DO search for similar open/closed preexisting issues before creating a new issue.**
```

Comment:
Please keep "DO"  this follows other Microsoft repos and helps brings attention to this line.